### PR TITLE
Report problem dialog

### PR DESCRIPTION
**Pull request by baruchiro**
For #57 

I started with two options:

- Button for opening **a new issue** with the content, like [this](https://github.com/baruchiro/israeli-bank-scrapers-desktop/issues/new?title=title&body=body)
- Button for opening the default **mail software** with the same data.

But opening the default mail software doesn't work well in Electron & Windows, so I started to implement an automatic mail sender in the app, but then I taught we don't need to keep our mind with using a mail, so I think I will implement a report sending with [Nucleus](https://www.nucleus.sh/nodejs/) or another alternative.

But I will do it in another PR.

Screenshots will upload soon.

### CODE REVIEW

File: src/components/MainPage.vue [hunk 1]
```diff
@@ -14,13 +14,17 @@
...
 
-      <v-btn
-        text
-        @click="() => openExternal('https://github.com/baruchiro/israeli-bank-scrapers-desktop/issues/new')"
-      >
-        <span class="mr-2">Report a problem</span>
-        <v-icon>mdi-chat-alert-outline</v-icon>
-      </v-btn>
+      <report-problem-dialog>
+        <template v-slot="{ on }">
```
Comment (baruchiro) [inline src/components/MainPage.vue]: I need to refactor this to pass the `activator` with `ref` and not with `template`

File: src/components/MainPage/LogSheet.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      type: String,
+      required: false,
+      default: null,
+    },
+  },
+  computed: {
+    textToShow() {
+      if (this.raw) return this.raw;
+
+      const lines = fs.readFileSync(this.$logger.transports.file.getFile().path).toString().split('\n');
```
Comment (baruchiro) [inline src/components/MainPage/LogSheet.vue]: I'm going to export this function to the `logger` itself

Comment (baruchiro) [inline src/components/MainPage/LogSheet.vue]: I'm going to export this function to the `logger` itself

**Comment by baruchiro** [comment]
image
image
image
image

(You can't write in Hebrew and open in Github, so I will remove this hint.

File: src/components/MainPage/ReportProblemDialog.vue [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+  const formattedLog = log ? `
+## Log
+\`\`\`
+${log}
+\`\`\`` : '';
+
+  return 'https://github.com/baruchiro/israeli-bank-scrapers-desktop/issues/new?'
+          + `title=${escape(title)}`
+          + `&body=${escape(formattedDetailes)}${escape(formattedLog)}`;
```
Comment (Arielgordon123) [inline src/components/MainPage/ReportProblemDialog.vue]:
```suggestion
          + `title=${encodeURI(title)}`
          + `&body=${encodeURIComponent(formattedDetailes)}${encodeURIComponent(formattedLog)}`;
```

(just for the record, the escape function is deprecated)

**Pull request merged**